### DISCUSSION TITLE
How to make fast refresh fast in design-system example?

### DISCUSSION THREAD

**Discussion by lorcan-codes** [vercel/turborepo#7530] [Help] [answered]
### Summary

The fast refresh in docs storybook works fine when we have a few components. But it gets slower as we have more components. The design system example does not explain how we can overcome the issue.

We should not have slow refreshes even if we have many components in ui package

To Reproduce
- Clone my repo: git clone https://github.com/lorcan-codes/my-design-system

- pnpm i in project root

- Run pnpm dev in root dir to start storybook

Make some changes in button component and it takes 5-10 seconds for changes to be visible
You can also watch this demo: https://drive.google.com/file/d/1hRToHYVyyxviHsMoBqUtgqlalLl7tnHv/view?usp=drive_link

### Example

https://github.com/lorcan-codes/my-design-system

**Comment by anthonyshew** [comment, marked as answer]
#7529
